manifest.json

```diff
@@ -115,7 +115,7 @@
 
      "QA": {
       "default": true,
-      "description": "Save all generated files (motion parameters, intermediate files), or just the fieldmap and any corrected timeseries",
+      "description": "Save a topup QA image comparing distorted to corrected images",
       "type": "boolean"
     }
```

Comment:
Do we want to restrict the input files to NifTIs?:
`
    "DWINegativeData": {
      "description": "Diffusion time series (Negative phase-encode, ie: L>>R or A>>P)",
      "base": "file",
      "type": {
        "enum": [
          "nifti"
        ]
      }
    },
`